content/en/docs/primer/definitions.md

```diff
@@ -0,0 +1,208 @@
+
+---
+
+### 🧰 Tool-Specific Considerations
+
+| **Tool**         | **Definition of Test**                                         |
+|------------------|----------------------------------------------------------------|
+| Jenkins          | Defined by pipeline steps that execute test commands (e.g., `mvn test`) |
+| GitHub Actions   | Typically a job or step that runs test commands or frameworks  |
+| GitLab CI/CD     | Often a dedicated job stage (e.g., `test` stage) with test execution commands |
+| Azure DevOps     | "Test Plans" or "Test Pipelines" that run automated tests and publish results |
+| SonarQube        | Primarily for static analysis and code quality, but often integrated with test results for coverage |
+| Jest/JUnit/Pytest| Frameworks for writing and executing tests within codebases    |
+| Selenium/Cypress | Frameworks for automated browser-based functional testing      |
+
+---
+
+## 💾 Store
+
+**Storing** is the automated process of persistently saving, managing, and
+making accessible any **artifact, data, or metadata** generated or consumed
+during the SDLC. It ensures that necessary components and information are
+retrievable for future use, auditing, or deployment.
+
+---
+
+### 📦 What Are “Stored Assets”?
+
+Stored assets encompass any durable output or input that is preserved across
+the SDLC. These include:
+
+-   **Build Artifacts**: Executables, container images, packages, libraries (e.g., JARs, WARs, Docker images, npm packages).
+-   **Source Code**: The versioned codebase managed in a Version Control System (VCS).
+-   **Configuration**: Environment-specific settings, deployment manifests, or infrastructure-as-code definitions.
+-   **Test Results**: Reports, logs, and metrics from test executions.
+-   **Logs**: Runtime logs from applications or infrastructure.
+-   **Metrics**: Performance, usage, or health data collected over time.
+-   **Documentation**: Generated or manually created documentation.
+-   **Snapshots/Backups**: Point-in-time copies of databases or systems.
+
+---
+
+### 🗄️ What Constitutes Storing?
+
+A typical automated storing process may include:
+
+-   ⬆️ **Publishing/Uploading**: Transferring an asset to a designated repository or storage location.
+-   🏷️ **Tagging/Versioning**: Assigning unique identifiers or metadata to stored assets for traceability.
+-   📦 **Indexing**: Cataloging assets within a repository for efficient searching and retrieval.
+-   🔐 **Access Control**: Applying permissions to regulate who can access or modify stored assets.
+-   🧹 **Retention/Cleanup**: Managing the lifecycle of stored assets, including deletion policies.
+-   🔄 **Replication**: Copying assets to multiple locations for redundancy or distribution.
+-   🔗 **Linking**: Establishing relationships between stored assets and other SDLC events (e.g., linking an artifact to the build that produced it).
+
```

Comment:
additional ideas:
- **Promotion**: Moving artifacts between environment-specific repositories (dev→staging→prod)
-   **Signing/Attestation**: Adding cryptographic signatures or metadata to stored artifacts
-   **Scanning**: Automated security/vulnerability scanning upon storage